accessibility/common-remediations/colours.md

```diff
@@ -17,12 +17,14 @@ Examples may include (but are not limited to):
 * Changing the background colour of an input field to red to indicate an error without also providing an error message
 * Changing the colour of an icon or text string to indicate some kind of status
 * Using a graph or a chart where colour is the only way to indicate different aspects of the data
+* Removing underlines from hyperlinks, leaving just colour to differentiate them from surrounding text
```

Comment:
ahhh fab thanks so much 🙏 I'd forgotten colour alone was included in this set of docs too. Could you explicitly add an "underline your links, dang it" resolution to the "How do I fix it?" section in this one also, please?

It'll be a bit redundant with it here and in the checklist, but if we ever pull the common remediations out into a standalone set of documents (i.e. into a bigger accessibility guide that's not crammed into the Playbook) then it'll be worth having it directly stated here as well.

Reply:
@hollsk Ah nice, yeah, will do.